Gemfile

```diff
@@ -49,6 +49,7 @@ gem 'config'
 gem 'dor-services-client', '~> 15.7' # For SDR datasets
 gem 'edtf'
 gem 'faraday'
+gem 'faraday-follow_redirects'
```

Comment:
Cool. I needed this for some of the UT comparison work I was doing and have it in a branch, it's helpful so glad to see it come in here.